### ISSUE TITLE
在数据预处理时，我们应该用源数据的平均值和标准差来初始化吗？

### ISSUE THREAD

**Issue by dayekuaipao** [tensor-yu/PyTorch_Tutorial#11]
或者是将其初始化到均值为0，方差为1？如果按照源数据的平均值和标准差来初始化的话输入数据会有什么区别呢？

**Comment by nanwei1** [comment]
> 或者是将其归一化到均值为0，方差为1？如果按照源数据的平均值和标准差来归一化的话输入数据和原来的数据会有什么区别呢？

For PyTorch you need to pass the mean and std dev of the training set as input arguments. Then "normalize" does the following: input = (input-mean)/std, so that the training data after normalisation will have zero mean and standard dev of 1.

From page 10 of the book:
细心的朋友可能会发现，在进行 Normalize 时，需要设置均值和方差，在这里直接给出了，但在实际应用中是要去训练集中计算的，天下可没有免费的午餐。这里给出计算训练集的均值和方差的脚本： /Code/1_data_prepare/1_5_compute_mean.py
